src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TO-DO-LIST</title>
+    <!-- <script src="https://kit.fontawesome.com/c41748354b.js" crossorigin="anonymous"></script> -->
```

Comment:
- [ ] [REQUIRED] Please, make sure that you don't use commented code in your code base because it makes the code base unclean and harder to maintain.